crates/common/mqtt_channel/src/topics.rs

```diff
@@ -6,13 +6,21 @@ use serde::Deserialize;
 use serde::Serialize;
 use std::collections::HashSet;
 use std::convert::TryInto;
+use std::fmt::Display;
+use std::fmt::Formatter;
 
 /// An MQTT topic
 #[derive(Debug, Clone, Eq, PartialEq, Deserialize, Serialize)]
 pub struct Topic {
     pub name: String,
 }
 
+impl Display for Topic {
+    fn fmt(&self, f: &mut Formatter) -> std::fmt::Result {
+        write!(f, "{}", self.name)
```

Comment:
One could avoid an extra layer of formatting.

```suggestion
        Display::fmt(&self.name, f)
```

Reply:
Resolved by 558238f0